pkg/compactor/bucket_compactor.go

```diff
@@ -361,6 +363,11 @@ func (c *BucketCompactor) runCompactionJob(ctx context.Context, job *Job) (shoul
 	uploadBegin := time.Now()
 	uploadedBlocks := atomic.NewInt64(0)
 
+	if err = verifyCompactedBlocksTimeRanges(compIDs, toCompactMinTime.UnixMilli(), toCompactMaxTime.UnixMilli(), subDir); err != nil {
+		level.Error(jobLogger).Log("msg", "compacted blocks verification failed", "err", err)
```

Comment:
nit: shall we mention that this error is currently ignored? (or change the log message to warning instead of error?)